internal/cli/atlas/backup/restores_start.go

```diff
@@ -91,41 +110,58 @@ func (opts *RestoresStartOpts) isAutomatedRestore() bool {
 	return opts.method == automatedRestore
 }
 
+func (opts *RestoresStartOpts) isPointInTimeRestore() bool {
+	return opts.method == pointInTimeRestore
+}
+
+func (opts *RestoresStartOpts) isDownloadRestore() bool {
+	return opts.method == downloadRestore
+}
+
 func (opts *RestoresStartOpts) validateParams() error {
 	if opts.clusterName == "" {
 		return errors.New("needs clusterName")
 	}
 
-	if !opts.isAutomatedRestore() {
-		if e := opts.automatedRestoreOnlyFlags(); e != nil {
-			return e
-		}
-	}
-
 	return nil
 }
 
-func (opts *RestoresStartOpts) automatedRestoreOnlyFlags() error {
-	if opts.oplogTS != 0 {
-		return fmt.Errorf(onlyFor, flag.OplogTS, automatedRestore)
+func markRequiredAutomatedRestoreFlags(cmd *cobra.Command) error {
+	if err := cmd.MarkFlagRequired(flag.TargetProjectID); err != nil {
+		return err
 	}
-	if opts.oplogInc > 0 {
-		return fmt.Errorf(onlyFor, flag.OplogInc, automatedRestore)
+
+	if err := cmd.MarkFlagRequired(flag.SnapshotID); err != nil {
+		return err
 	}
-	if opts.pointInTimeUTCMillis > 0 {
-		return fmt.Errorf(onlyFor, flag.PointInTimeUTCMillis, automatedRestore)
+
+	if err := cmd.MarkFlagRequired(flag.TargetClusterID); err != nil {
+		return err
 	}
-	return nil
+
+	return cmd.MarkFlagRequired(flag.ClusterName)
 }
 
-func markRequiredAutomatedRestoreFlags(cmd *cobra.Command) error {
+func markRequiredPointInTimeRestoreFlags(cmd *cobra.Command) error {
 	if err := cmd.MarkFlagRequired(flag.TargetProjectID); err != nil {
 		return err
 	}
 
+	if err := cmd.MarkFlagRequired(flag.TargetClusterID); err != nil {
+		return err
+	}
+
 	return cmd.MarkFlagRequired(flag.ClusterName)
 }
 
+func markRequiredDownloadRestoreFlags(cmd *cobra.Command) error {
+	if err := cmd.MarkFlagRequired(flag.SnapshotID); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
[nit this is the same as]
```suggestion
	return cmd.MarkFlagRequired(flag.SnapshotID)
```
which brings the question, do you need this function?